R/utils.R

```diff
@@ -1,14 +1,18 @@
-#' Parse API answer.
+#' Parse API answer
 #'
+#' Returns data.frame from parsed xml API answer.
+#'
+#' @param api_answer parsed xml API answer
 #' @param multiple If multiple results match your query, should the function
 #' recursively fetch data for each of one of them (`multiple = TRUE`) or
 #' return a data.frame containing only titles and ISSN of all matches
 #' (`multiple = FALSE`)
+#' @inheritParams check_key
 #'
 #' @keywords internal
 #'
 #' @import xml2
-parse_answer = function(api_answer, multiple = FALSE) {
+parse_answer = function(api_answer, multiple = FALSE, key = NULL) {
```

Comment:
I not super happy with sending the key param back and forth between function but I haven't thought too much about it so maybe there is no other way...

Anyways, if you want to do this, you need to pass the `key` param when you call `parse_answer` in `rr_journal_issn` or `rr_journal_name`.

Reply:
Yep right... I know, it gets difficult to track the key parameter in this case.
However, it's a pity not to use it if the user has defined a key.
Especially when retrieving multiple journals.